frontend/src/assets/css/variables.scss

```diff
@@ -1,49 +1,97 @@
 :root {
-    // Error colors
-    --error-main: #ff2121;
+    // *Basic Colors*
 
-    // Action colors
-    --action-hover: #2e61ae;
+    // Red
+    --torch-red: #ff2121;
+
+    // Azure
+    --azure: #2e61ae;
+
+    // White
+    --white: #ffffff;
+    --white-lilac: #f7f8fc;
+
+    // St Tropaz
+    --st-tropaz: #274f8d;
+
+    // Gray
+    --steel-gray: #1f1e29;
+    --slate-gray: #7c8798;
+
+    // Nepal
+    --nepal: #90a3bf;
+
+    // Botticelli
+    --botticelli: #d5dce8;
+
+    // Solitude
+    --solitude: #dce5ff;
+
+    // Mountain Meadow
+    --mountain-meadow: #12b76a;
+
+    // Porcelain
+    --porcelain: #f5f6f7;
+
```

Comment:
I am not sure about these comments. It seems as if the information is duplicated.

Reply:
we can have one source for our color naming by  https://chir.ag/projects/name-that-color/#F5F6F7 and reuse basic variables in other variables without duplication.

Reply:
Sorry. I was talking about these comments // Porcelain. You have //porcelain and then variable name --porcelain.

Reply:
as for me it's more readable, but maybe it's too much for your project